frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeader.tsx

```diff
@@ -13,9 +10,15 @@ interface IConstraintAccordionViewHeaderProps {
     onDelete?: () => void;
     onEdit?: () => void;
     onUse?: () => void;
+    /**
+     * @deprecated
+     */
     singleValue: boolean;
```

Comment:
Why don't we just delete these entirely? They don't do anything anymore, right? Both this and the next

Reply:
Yup, but this can be done in a separate PR. It tried to make this one as simple as possible, only changing things that didn't pass tests/tsc.